db/migrate/20240424214558_add_locking_columns.rb

```diff
@@ -0,0 +1,7 @@
+class AddLockingColumns < ActiveRecord::Migration[7.0]
+  def change
+    add_column :episodes, :lock_version, :integer, null: false, default: 0
+    add_column :feeds, :lock_version, :integer, null: false, default: 0
+    add_column :podcasts, :lock_version, :integer, null: false, default: 0
```

Comment:
This is the only change - no null lock_versions.

To QA this locally, you can:

1. Browse to an episode with no media
2. Open `psql` and manually make the lock_version null
    ```
    feeder_development=# ALTER TABLE episodes ALTER COLUMN lock_version DROP NOT NULL;
    ALTER TABLE

    feeder_development=# UPDATE episodes SET lock_version = NULL WHERE guid = '<your_guid>';
    UPDATE 1
    ```
3. In your browser, try to upload and save a media file. See the server return 409 Conflict
4. Alter another field such as segment count and upload the media file again. Save succeeds.
5. Delete audio file and save.
6. Set the NOT NULL again from psql:
    ```
    feeder_development=# UPDATE episodes SET lock_version = 0 WHERE guid = '<your_guid>';
    UPDATE 1

    feeder_development=# ALTER TABLE episodes ALTER COLUMN lock_version SET NOT NULL;
    ALTER TABLE
    ```
7. Retest just uploading a file as in (3).  See that it succeeds this time.